admin/acceptance_tests/left_menu_test.py

```diff
@@ -29,11 +29,6 @@ class TestLeftMenu(AbstractViewsTests):
 
     _prefix = '/roles'
 
-    def test_404(self, test_app):
```

Comment:
even if the page does not exist, were were used to have nav bar displayed... It still work so with demo.

Reply:
Yes, but we do not want admin 404 with navbar displayed for non admin users.

Reply:
Would be cool to have the admin 404 with navbar in case we are in /admin, but this should be handled in general not_found_view of the scaffold app.

Reply:
It may be possible to register a /admin/* route (to be verified) but will not handle the case a NotFound exception is raised when a record does not exist.

Reply:
could be cool to provide a helper to render the admin 404 in global not_found_view.

Reply:
For me we should target this point later, I relay don't want that interfere with the main application...